liferay-echo/liferay-portal
# LPS-121883 Migrate management toolbar v2 usages in layout/layout-admin-web

Attached is an update for http://issues.liferay.com/browse/LPS-121883.  

**Test Plan:**
1.  Login to Liferay
2. Click on the **Control Menu**
3. Click on the **Page Tree** link
4. Click on the **Gear Icon**
5. Under the **Look And Feel** panel, click on the **Change Current Theme** button
6. See Management Toolbar in popup window
7. Go back to step 3 and now click on the `+` icon
8. Click on **Add Collection Page** from menu
9. See Management Toolbar
10. Go back to step 3 and now click on the **Public Pages** link in the page tree
11. See Management Toolbar
![layout](https://user-images.githubusercontent.com/860951/109217376-a1386b00-776a-11eb-9345-384b066bc0a4.gif)
